lambdas/commands.py

```diff
@@ -114,8 +117,23 @@ def _etl_v1_generate_transform_commands_method(
 
 
 # NOTE: FEATURE FLAG: branching logic + method removed after v2 work is complete
-def _etl_v2_generate_transform_commands_method() -> dict[str, list[dict]]:
-    raise NotImplementedError
+def _etl_v2_generate_transform_commands_method(
+    extract_output_files: list[str],
+    input_data: dict,
+    timdex_bucket: str,
+    run_id: str,
+) -> dict[str, list[dict]]:
+    files_to_transform: list[dict] = []
+    source = input_data["source"]
+    for extract_output_file in extract_output_files:
+        transform_command = [
+            f"--input-file=s3://{timdex_bucket}/{extract_output_file}",
+            f"--output-location=s3://{timdex_bucket}/dataset",
```

Comment:
This is subtle but important: this effectively sets `/dataset` as _the_ TIMDEX parquet dataset.

We may well want to explore setting this via an SSM param / env var at some point, but I'd propose to keep this hardcoded at the moment.  We did similar things with `source`, where each application knew that the `source` at hand was the folder to look for in S3.  Now, it's kind of a simplication where everyone just knows that `/dataset` is where the dataset is located.

Reply:
Just to recap, this will result in the following file tree: 

```
└── timdex-bucket/
    └── dataset/
        └── year=2024/
            └── month=12/
                └── day=01/
                    └── <uuid>-<optional-sequence>.parquet
```

where UUID uniquely identifies a _run per source_?